packages/sampleProp.conf

```diff
@@ -1,4 +1,4 @@
-#### [DEFAULT] ####
+[DEFAULT]
```

Comment:
Do we require this to be removed from other sections as well ?, #### [IOTCC] ####
, #### [GENERICMQTT] ####